### ISSUE TITLE
deflare plots squished using conda

### ISSUE THREAD

**Issue by kglotfelty** [cxcsds/ciao-contrib#751]
I'm guessing it's a matplotlib diff between ciao-install vs. conda 

```bash
% deflare infile="/data/sdsreg/current_regtest/input/deflare/acis3103_500s.lc" \
  outfile="3103.gti" method="sigma" nsigma="3" \
  plot="yes" save="goo.png" rateaxis="y" pattern="solid" \
  good_color="lime" exclude_color="red" minlength="3" mean="0" \
  stddev="0" scale="1.2" minfrac="0.1" verbose="1"

...stuff...

Light curve cleaned using the lc_sigma_clip routine.
/export/miniconda/envs/ciao-4.15/bin/deflare:153: UserWarning: Tight layout not applied. tight_layout cannot make axes height small enough to accommodate all axes decorations.
  plt.savefig(oplot, format=format)
Created: goo.png
Press ENTER to close the plot window and exit:/export/miniconda/envs/ciao-4.15/bin/deflare:122: UserWarning: Tight layout not applied. tight_layout cannot make axes height small enough to accommodate all axes decorations.
  input("Press ENTER to close the plot window and exit:")
```
creates this using ciao-4.15 installed w/ conda

![goo](https://user-images.githubusercontent.com/6662179/228333565-7f68474d-2fc7-44b6-a0e4-c871bccd2c19.png)

```bash
% conda list  | grep mat
matplotlib                3.6.2           py310hff52083_0    conda-forge
matplotlib-base           3.6.2           py310h8d5ebf3_0    conda-forge
matplotlib-inline         0.1.6                    pypi_0    pypi
nbformat                  5.7.0                    pypi_0    pypi
```

with the ciao-install version the plot looks correct (fills the image).

**Comment by nplee** [comment]
fix in #756 tested on Matplotlib 3.5, 3.6, and 3.7.

**Comment by kglotfelty** [comment]
This is fixed with #756
